test/apps/iodemo/ucx_wrapper.cc

```diff
@@ -1195,6 +1201,15 @@ void UcxConnection::established(ucs_status_t status)
 
     _context.remove_connection_inprogress(this);
     invoke_callback(_establish_cb, status);
+
+    if (status == UCS_OK) {
+        while (!_iomsg_recv_backlog.empty()) {
+            const UcxContext::iomsg_buffer_t &iomsg =
+                    _iomsg_recv_backlog.back();
+            _context.dispatch_io_message(this, &iomsg[0], iomsg.size());
+            _iomsg_recv_backlog.pop_back();
```

Comment:
the ordering is changed here, is it importand for iodemo? maybe use std::queue?

Reply:
fixed